### ISSUE TITLE
Qalculate - add params for installing only Qt/GTK version

### ISSUE THREAD

**Issue by Destroy666x** [C4illin/Choco-Packages#5]
Not sure if possible with installer args or if different installers are necessar, but would be nice if it was possible to install only e.g. Qt version of Qalculate.

**Comment by C4illin** [comment]
I don't think that would be possible since they are packaged together. The option would be to delete one of the versions after finished installation but that seems odd
